DATA SCIENCE-UC00201_Tourist_Hotspot_Vs_Hospitality_Capacity- 100% complete Sprint 2

manya0033 opened the pull request:
Updated with the following features after Sprint 1:
Interactive Folium maps with capacity gap visualisation and venue density heatmaps.
Smart dining app simulation with real-time recommendation interface.
Interactive recommendations map with priority actions and timelines and final conclusion.

ajaiysentil reviewed (changes requested):
UC00201 Tourist Hotspot vs Hospitality Capacity Review 

1. Code Quality & Readability –

The Notebook is well organised and structured,  variable names are named appropriately.  Code quality, is good. 

2. Use case naming conventions - 

The use case naming convention is followed appropriately 

3. Location of the pull request - 

The location of the pull request is under the playground folder within a file with students name. 
This has been done correctly 

4. Language - 
There is a lot of US English utilised within the notebook, key terms such as vizualise, analyze and utilize. 
Make sure to go through the document and amend these to AUS English.  


5. Tutorial - 
The notebook is well structured and easy to read, titles are utilised for each key section. Each section is well described, provided clear analysis on insights from the project. 
Code is well written and has comments. 
All the visualisations are implemented well with a  title and clear labelling, with key visualisation provided with detailed description of the observations. 
Each section is clearly structured and findings are communicated effectively.   

6. Template - 

The template has been followed correctly 

7. API - 

The data is pulled from the Melbourne Open Data site using the API.

8. API Key - 

API key is not visible 


Great work on the Tourist Hotspot vs Hospitality Capacity use case, you have successfully analysed Melbourne’s tourist attractions and hotspots to discover key imbalances. The code was well written, all descriptions were clear and detailed. As Jenny stated in her review, my only amendment is for the Language please change the US English to AU English.

ajaiysentil reviewed (approved):
great work updating the required changes

jennynguyen2112 commented:
Hi @manya0033, thank you for addressing the comments. Please also correct some remaining words in US English: utilization, individualized, underutilized.
Additionally, please also remove irrelevant file "Untitled17.jpynb" from the folder. Thank you Manya!

manya0033 commented:
> Hi @manya0033, thank you for addressing the comments. Please also correct some remaining words in US English: utilization, individualized, underutilized. Additionally, please also remove irrelevant file "Untitled17.jpynb" from the folder. Thank you Manya!

Thank you Jenny, just uploaded the corrected one and opening another PR to delete the untitled one!

jennynguyen2112 reviewed (approved):
Thank you Manya! Please proceed to merge.

manya0033 commented:
> great work updating the required changes

thank you!!
